src/operations/attest_key.rs

```diff
@@ -25,6 +25,16 @@ pub enum Operation {
         /// Name of key to be used for attesting
         attesting_key_name: Option<String>,
     },
+    ///Key and platform attestation
+    CertifyAndQuote {
+        /// Name of key to be attested
+        attested_key_name: String,
+        /// The nonce to be used in the TLS handshake
+        #[derivative(Debug = "ignore")]
+        nonce: Zeroizing<Vec<u8>>,
```

Comment:
The nonce isn't actually secret or sensitive, so you can just leave it as `Vec<u8>` here.